### PR TITLE
Upgrade to Spring Boot 1.4.1.RELEASE

### PR DESCRIPTION
**Pull request by gregturn**
NOTE: Due to API changes in Tomcat 8.5 and Kork, Tomcat is pinned to 8.0

### CODE REVIEW

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/DisableTitusServerGroupAtomicOperation.groovy [hunk 1]
```diff
@@ -19,7 +19,7 @@ import com.netflix.spinnaker.clouddriver.titus.TitusClientProvider
 import com.netflix.spinnaker.clouddriver.titus.deploy.description.EnableDisableServerGroupDescription
 
 class DisableTitusServerGroupAtomicOperation extends AbstractEnableDisableAtomicOperation {
-  static final String phaseName = "DISABLE_TITUS_SERVER_GROUP"
+  final String phaseName = "DISABLE_TITUS_SERVER_GROUP"
```
Comment (cfieber) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/DisableTitusServerGroupAtomicOperation.groovy]: this doesn't seem necessary

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/EnableTitusServerGroupAtomicOperation.groovy [hunk 2]
```diff
@@ -19,7 +19,7 @@ import com.netflix.spinnaker.clouddriver.titus.TitusClientProvider
 import com.netflix.spinnaker.clouddriver.titus.deploy.description.EnableDisableServerGroupDescription
 
 class EnableTitusServerGroupAtomicOperation extends AbstractEnableDisableAtomicOperation {
-  static final String phaseName = "ENABLE_TITUS_SERVER_GROUP"
+  final String phaseName = "ENABLE_TITUS_SERVER_GROUP"
```
Comment (cfieber) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/EnableTitusServerGroupAtomicOperation.groovy]: also doesn't seem necessary

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/CacheResultBuilder.groovy [hunk 3]
```diff
@@ -37,7 +37,7 @@ class CacheResultBuilder {
   }
 
   NamespaceBuilder namespace(String ns) {
-    namespaceBuilders.get(ns)
+    namespaceBuilders[ns]
```
Comment (cfieber) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/CacheResultBuilder.groovy]: this is semantically different than the original

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy [hunk 4]
```diff
@@ -17,9 +17,12 @@
 package com.netflix.spinnaker.clouddriver.aws.cache
 
 import com.netflix.frigga.Names
-import static com.netflix.spinnaker.clouddriver.aws.AmazonCloudProvider.AWS
+import com.netflix.spinnaker.clouddriver.aws.AmazonCloudProvider
 
 class Keys {
+
+  static def AWS = AmazonCloudProvider.AWS
```
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy]: this shouldn't need to change?

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/CacheResultBuilder.groovy (hunk 3 above)
Comment (gregturn) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/CacheResultBuilder.groovy]: Reverted.

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/DisableTitusServerGroupAtomicOperation.groovy (hunk 1 above)
Comment (gregturn) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/DisableTitusServerGroupAtomicOperation.groovy]: Given the parent is non-static, I think this is a valid fault.

```
Error:(22, 3) Groovyc: The method 'java.lang.String getPhaseName()' is already defined in class 'com.netflix.spinnaker.clouddriver.titus.deploy.ops.DisableTitusServerGroupAtomicOperation'. You cannot have both a static and an instance method with the same signature
```

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/EnableTitusServerGroupAtomicOperation.groovy (hunk 2 above)
Comment (gregturn) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/EnableTitusServerGroupAtomicOperation.groovy]: Same as earlier.

File: clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/DisableTitusServerGroupAtomicOperation.groovy (hunk 1 above)
Comment (cfieber) [inline clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/ops/DisableTitusServerGroupAtomicOperation.groovy]: makes sense, thanks!

File: clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy (hunk 4 above)
Comment (cfieber) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy]: I'd be more in favor of the AmazonCloudProvider change to make that constant public

Comment (gregturn) [inline clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/cache/Keys.groovy]: Done.

**Pull request merged**